ui/src/styles/backtest.css

```
@@ -0,0 +1,510 @@
+  flex-direction: column;
+  gap: var(--sp-3);
+}
+
+.bt-twin {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)